ykrt/src/compile/jitc_yk/jit_ir/mod.rs

```diff
@@ -1844,7 +1856,7 @@ impl Inst {
         Ok(inst)
     }
 
-    /// Returns the size of the local variable that this instruction defines (if any).
+    /// Returns the size in bytes of the local variable that this instruction defines (if any).
```

Comment:
Should the function be called `def_bytew` for consistency?

Reply:
I would actually prefer it to be just `bytew` but I have several renaming PRs to do :)